test/e2e/performance_test.go

```diff
@@ -83,6 +88,14 @@ func BenchmarkRealizeNetworkPolicy(b *testing.B) {
 	}
 }
 
+func BenchmarkRealizeCNP(b *testing.B) {
```

Comment:
It's cool to have a benchmark but I am not convinced we should have added it in this PR. BTW, I may have missed it, but do we ensure that CNP is enabled here? If so, how?

Reply:
Thanks for the catch.... The benchmark test must have passed on my local kind cluster since I had cnp e2e tests which enabled the feature. Removing it from this PR and will revisit how it can fit in to the new test proposal for #893